support/metrics/metricsTracker.go

```diff
@@ -228,33 +257,37 @@ func (mt *MetricsTracker) SendUpdateEvent(now time.Time, success bool, millisFor
 
 // SendDeleteEvent sends the delete Amplitude event.
 func (mt *MetricsTracker) SendDeleteEvent(exit bool) error {
-	commonProps := mt.props
-	commonProps.SecondsSinceStart = time.Now().Sub(mt.botStartTime).Seconds()
+	mt.props.SecondsSinceStart = time.Now().Sub(mt.botStartTime).Seconds()
```

Comment:
same here -- mt.props vs commonProps -- whatever solution we decide to move forward with